### ISSUE TITLE
Ability to tag datasets that have been imported via etl

### ISSUE THREAD

**Issue by spoonerf** [owid/owid-grapher#1869]
## Core problem

Typically our datasets and charts are tagged with a topic and then all of the datasets and charts of a given topic can be viewed together e.g. [here](https://owid.cloud/admin/tags/99). It is useful to be able to see all the charts and datasets of a given topic together and will be especially useful as we go through all the topic page redesigns. 

Currently it is not possible to tag a dataset that has been imported via etl or importers. There are 597 such datasets, listed [here](https://owid-datasette-y43qr.ondigitalocean.app/owid?sql=select%0D%0A++id%2C+name%0D%0Afrom%0D%0A++datasets%0D%0Awhere+namespace+%21%3D+%27owid%27%0D%0Aand%0D%0A++id+not+in+%28%0D%0A++++select%0D%0A++++++datasetId%0D%0A++++from%0D%0A++++++dataset_tags%0D%0A++%29%0D%0A). 

## Proposed solution

Allow for the tagging of all datasets, regardless of their input method. However, I am not certain how possible or easy this is?

**Comment by danyx23** [comment]
Sounds good.

@pabloarosado @Marigold do you think it is fine to always change this information in the admin UI? Or do you think it should be possible to set tags from the ETL? If the latter, how should we manage this so that there is a clear source of truth?

**Comment by Marigold** [comment]
@spoonerf [mentioned](https://owid.slack.com/archives/C0193RW5E2J/p1673951663157689?thread_ts=1673950976.816799&cid=C0193RW5E2J) it'd be more useful in grapher

> I think it would be most useful if tags could be edited in admin - particularly as it might need to be done retrospectively to some bulk uploads. But if that’s not possible then adding it to the grapher steps would be good!

Setting tags in ETL would be more straightforward (we'd just add them to metadata and propagate to DB). I'm a bit worried that making only tags editable for automatically uploaded datasets would require significant changes to admin. I'll take a look how much effort could it be.

**Comment by spoonerf** [comment]
Thanks for taking a look into this! @edomt had a suggestion that perhaps the dataset tags could be automatically created from the tags of the charts in the dataset - which might save a lot of manual tagging!

**Comment by Marigold** [comment]
It hasn't been fixed yet (no one asked since then). It hasn't been decided whether we should be editing tags in admin or in ETL or automatically get them from charts as Fiona suggested. Perhaps we can close this or change to "nice to have"?

**Comment by edomt** [comment]
Yes, I'd say this is a "nice to have" – dataset tagging isn't as central to our system as chart tagging.
